src/pages/swapForm.tsx

```diff
@@ -239,9 +228,9 @@ export const SwapForm = () => {
 		// dispatch({ type: SourceEnum.NETWORK, payload: 'ETH' });
```

Comment:
maybe I misunderstood this block. Why do we leave commented code in the file?

Reply:
You are right, usually we don't do this (and we shouldn't). There is two parts with longer commented code blocks: this one and the one for the settings modal. 
Since this one has been commented for a long time now I decided to remove it, the other one stays because we need it. The only thing that is missing for us to use it is the backend API to store the email addresses. @Erenshtein any update on this one? Did you have time already to setup an API that stores the email address of a user if he signs up? I guess it wasn't top priority so far

Reply:
@marcuszierke I need to check the email form with Mario since a few email-related topics were raised already.